data/processed/hesa/total_postgraduates.csv

```diff
@@ -0,0 +1,201 @@
+,year,nuts_id,nuts_year_spec,total_postgraduates
+0,2014/15,UKH2,2018,1005.0
```

Comment:
I think `year` is supposed to be an `int` so we should probably choose the start or the end of the academic year? (*)

Reply:
todo